pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs

```diff
@@ -217,33 +302,23 @@
                       <!-- ################################################## -->
 
                       <% if (authz_data.has_instructor_edit) { %>
-                      <a class="dropdown-item" onclick="$('#row<%= iRow %>PopoverRegrade').popover('show')" href="#">
-                        <i class="fa fa-sync" aria-hidden="true"></i> Regrade
-                      </a>
-                      <% } %>
-
-                      <!-- ################################################## -->
-
-                      <% if (authz_data.has_instructor_edit) { %>
-                      <a class="dropdown-item <% if (row.open) { %>disabled<% } %>" <% if (!row.open) { %>onclick="$('#row<%= iRow %>PopoverOpen').popover('show')"<% } %> href="#">
-                        <i class="far fa-circle" aria-hidden="true"></i> Re-open
+                      <a class="dropdown-item <% if (!row.open) { %>disabled<% } %>" <% if (row.open) { %>onclick="$('#row<%= iRow %>PopoverClose').popover('show')"<% } %> href="#">
```

Comment:
Ditto for accessibility concerns - this should be a `<button type="button">` (apologies, but GitHub won't let me leave a code suggestion here).